clang/lib/CIR/Dialect/Transforms/TargetLowering/LowerFunction.cpp

```diff
@@ -1030,6 +1049,32 @@ mlir::Value LowerFunction::rewriteCallOp(const LowerFunctionInfo &CallInfo,
 
       break;
     }
+    case ABIArgInfo::Indirect:
+    case ABIArgInfo::IndirectAliased: {
+      assert(NumIRArgs == 1);
+      // TODO(cir): For aggregate types
+      // We want to avoid creating an unnecessary temporary+copy here;
+      // however, we need one in three cases:
+      // 1. If the argument is not byval, and we are required to copy the
+      // 2. If the argument is byval, RV is not sufficiently aligned, and
+      //    source.  (This case doesn't occur on any common architecture.)
+      //    we cannot force it to be sufficiently aligned.
+      // 3. If the argument is byval, but RV is not located in default
+      //    or alloca address space.
+      cir_cconv_assert(!::cir::MissingFeatures::skipTempCopy());
+
+      mlir::Value Alloca = findAlloca(I->getDefiningOp());
```

Comment:
all variables name should be lowercase: https://llvm.github.io/clangir/GettingStarted/coding-guideline.html, this PR has a mix of them, please fix that (fine to leave the ones that are already wrong as is, but let's not introduce new ones)

Reply:
updated